pkgs/servers/audiobookshelf/generate.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix -p prefetch-npm-deps
+
+src="$(nix-build . -A audiobookshelf.src --no-out-link)"
+
+cd "$(dirname "${BASH_SOURCE[0]}")" || exit 1
+
+hash=$(prefetch-npm-deps "$src/client/package-lock.json")
+echo "npmDepsHash = \"$hash\";" > test.txt
```

Comment:
Is that file even used?

Reply:
Nope, that was left over from debugging the sed command